.github/workflows/compile_test.yaml

```diff
@@ -275,29 +278,32 @@ jobs:
           cd ${GITHUB_WORKSPACE}/docs/demos/ecp_feb_2023
           make all
       - name: Test DYAD with separate FS c
-        timeout-minutes: 1
+        timeout-minutes: 2
```

Comment:
Was this necessary? If it needs two minutes to complete, something is going on.

Reply:
Can you run it with debug output enabled?

Reply:
Fixed now. Have tested on my local repo.
https://github.com/wangvsa/dyad/actions/runs/15485596984/job/43599461514
We can merge this now. @JaeseungYeom